static/js/components/ProfileImage.js

```diff
@@ -1,24 +1,37 @@
 // @flow
 /* global SETTINGS: false */
 import React from 'react';
+import Icon from 'react-mdl/lib/Icon';
 
 import { makeProfileImageUrl, getPreferredName } from '../util/util';
 import type { Profile } from '../flow/profileTypes';
 
 export default class ProfileImage extends React.Component {
   props: {
-    profile: Profile
+    profile: Profile,
+    editable: boolean
+  };
+
+  static defaultProps = {
+    editable: false
   };
 
   render () {
-    const { profile } = this.props;
+    const { profile, editable } = this.props;
     const imageUrl = makeProfileImageUrl(profile);
+    const cameraIcon = editable => (
+      editable ? <span className="img"><Icon name="camera" /></span> : null
+    );
```

Comment:
also can you remove the `<span>` tag rapping the `Icon`? If you want to style the icon it would be better to just use a selector that targets it, i.e. you could put `.camera-icon` on it or something.